server/build.gradle

```diff
@@ -35,6 +35,9 @@ dependencies {
     }
     implementation group: 'io.dropwizard.metrics', name: 'metrics-core', version: '4.2.2'
     implementation group: 'io.dropwizard.metrics', name: 'metrics-jmx', version: '4.2.2'
+    implementation group: 'io.prometheus', name: 'simpleclient_dropwizard', version: '0.11.0'
+    implementation group: 'io.prometheus', name: 'simpleclient_servlet', version: '0.11.0'
+    implementation group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '9.4.43.v20210629'
```

Comment:
jetty-servlet seems to have newer versions but any particular reason for using the specified version?

Reply:
Thank you for taking a look at this!

According to the following doc, Jetty `9.4.x` is for Java 8, and `10.0.x `and `11.0.x` is for Java 11:
https://www.eclipse.org/jetty/

As Scalar DB supports Java 8 at the moment, I decided to use `9.4.43.v20210629` that's the latest version in `9.4.x`.